### ISSUE TITLE
Dorado 'Failed to open pod5'

### ISSUE THREAD

**Issue by femiliani** [nanoporetech/dorado#46]
Trying to run dorado with pod5 gives me this error: `Failed to open file ./pod5/output.pod5: IOError: Unknown embedded file type`

I converted my fast5 to pod5 with:
`pod5-convert-from-fast5 fast5_pass/ pod5`
When running dorado (new binary) with the pod5 i get the error (full output below). Running it with fast5 works fine, and inspecting the pod5 with `pod5-inspect` throws no errors. 

Thanks for the help! 
-Francesco 

```
/home/f002sd4/ont-dependencies/dorado/bin/dorado basecaller  /home/f002sd4/ont-dependencies/dorado/dna_r10.4.1_e8.2_400bps_hac@v3.5.2 ./pod5/ -x "cuda:0" --remora-models /home/f002sd4/ont-dependencies/dorado/dna_r10.4.1_e8.2_400bps_hac@v3.5.2_5mCG@v2
[2022-11-11 10:48:35.417] [info] > Creating basecall pipeline
@HD     VN:1.6  SO:unknown
@PG     ID:basecaller   PN:dorado       VN:0.0.2+acbca36        CL:dorado basecaller /home/f002sd4/ont-dependencies/dorado/dna_r10.4.1_e8.2_400bps_hac@v3.5.2 ./pod5/ -x cuda:0 --remora-models /home/f002sd4/ont-dependencies/dorado/dna_r10.4.1_e8.2_400bps_hac@v3.5.2_5mCG@v2
[2022-11-11 10:48:39.807] [error] Failed to open file ./pod5/output.pod5: IOError: Unknown embedded file type
[2022-11-11 10:48:39.807] [error] Failed to query batch count: Invalid: null file passed to C API
[2022-11-11 10:48:40.206] [info] > Reads basecalled: 0
[2022-11-11 10:48:40.206] [info] > Samples/s: 0.000000e+00
[2022-11-11 10:48:40.206] [info] > Finished
```

**Comment by iiSeymour** [comment]
Hey @femiliani 

Did you just convert this file with the latest release (0.0.41)? If so we need to upgrade our pod5 dependency to the latest version. I will sort this on Monday,

Regards

**Comment by femiliani** [comment]
@iiSeymour  I did indeed use the latest pod5 release, thanks for catching the problem, really excited to see where dorado is headed! Enjoy the weekend!

**Comment by iiSeymour** [comment]
Thanks for flagging this @femiliani it really should have been supported in v0.0.2 🤦🏻 

Resolved with [v0.0.3](https://github.com/nanoporetech/dorado/releases/tag/v0.0.3).
